### PR TITLE
README: Document changes with key steps in first page itself.

### PR DESCRIPTION
**Pull request by amarts**

### CODE REVIEW

File: doc/README.adoc [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+
+== Talks and Blog posts
+
+1. [Blog] link:https://medium.com/@tumballi/glusters-management-in-k8s-13020a561962[Gluster��s management in k8s]
+2. [Blog] link:https://aravindavk.in/blog/gluster-and-k8s-portmap/[Gluster and Kubernetes - Portmap]
+3. [Talk] link:rethinking-gluster-management-using-k8s.pdf[Rethinking Gluster Management using k8s]
+4. [Demo] link:https://asciinema.org/a/259949[Asciinema recording - Kadalu Setup]
+5. [Demo] link:https://asciinema.org/a/259951[Asciinema recording - KaDalu CSI to claim Persistent Volumes]
+
```
Comment (leelavg) [inline doc/README.adoc:11]: - Will it be good to mention this as well https://thoughtexpo.com/tags/kadalu/?
- No problem whatsoever if it's not mentioned

Comment (amarts) [reply doc/README.adoc:11]: Sure... makes sense to add.

File: doc/create-pvc.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (aravindavk) [inline doc/create-pvc.adoc:1]: Add direct link to this chapter here https://github.com/kadalu/kadalu/blob/devel/doc/index.yml

File: README.md [hunk 3]
```diff
@@ -7,28 +7,36 @@
...
+[Kadalu](https://kadalu.io) is a project to provide Persistent Storage in container ecosystem (like kubernetes, openshift, RKE, etc etc). Kadalu operator deploys CSI pods, and **gluster storage** pods as per the config. You would get your PVs served through APIs implemented in CSI.
 
-Try it in few minutes to understand more!
+## Get Started
 
-## Documentation
+Getting started is made easy to copy paste the below commands.
 
-Start with our [Quick Start Guide](doc/quick-start.adoc). More documentation is at [`doc/` folder](doc/).
+[source,console]
```
Comment (leelavg) [inline README.md]: I think either we need to change README.md to README.adoc or replace adoc variants in this file with markdown

Comment (amarts) [reply README.md]: Ack!

File: doc/create-pvc.adoc (hunk 2 above)
Comment (amarts) [reply doc/create-pvc.adoc:1]: This I am not sure if we need it at all... If someone checks, its a standard step of asking a PVC. should we still have it?

**Review by leelavg — APPROVED** [review]
LGTM

File: doc/create-pvc.adoc (hunk 2 above)
Comment (aravindavk) [reply doc/create-pvc.adoc:1]: No. Only if topic required in the side menu in the website

**Pull request merged**